static/js/app.js

```diff
@@ -532,3 +532,12 @@ function vpnBannerLogic() {
 
   vpnPromoFunctions.init();
 }
+
+//Micro Survey Banner
+function dismissSurvey() {
+	var survey_banner = document.getElementById("micro-survey-banner");
+	survey_banner.classList.toggle("hidden");
```

Comment:
Instead of `.hidden`, let's use state-based classes names, and use `.is-hidden`.  (You'll need to add a `.is-hidden` class to the SCSS)

From [Protocol's documentation](https://protocol.mozilla.org/docs/naming.html): 
> is- to indicate a current state, e.g. .mzp-is-active, .mzp-is-collapsed (typically assigned by JS and not hard-coded).

Note – as an example, if an element already has a ".hidden" classes added, we would want to change the default behavior to be hidden and use a `.is-visible`on the JS toggle.